web-app/js/portal/data/MetadataRecordFetcher.js

```diff
@@ -40,7 +38,7 @@ Portal.data.MetadataRecordFetcher = Ext.extend(Ext.util.Observable, {
 
             if (metadataRecord) {
                 _this.dataCollectionStore.add(dataCollection);
-                Ext.MsgBus.publish(PORTAL_EVENTS.VIEW_DATA_COLLECTION, dataCollection);
+                viewport.setActiveTab(TAB_INDEX_VISUALISE);
```

Comment:
Not sure how I feel about adding a dependency on this global... I'm ok to merge this for now though, having said that.